rules/civilian-props.yaml

```diff
@@ -42,12 +42,74 @@ camisc04:
 		Footprint: x
 		Dimensions: 1, 1
 	Tooltip:
-		Name: Mail Box
+		Name: Mailbox
 	Health:
 		HP: 10
 	Armor:
 		Type: Concrete
 
+camisc05:
+	Inherits: ^CivBuilding
+	Building:
+		Footprint: x
+		Dimensions: 1, 1
```

Comment:
This shouldn't be needed as `^CivBuilding` inherits `^Building` and this has [this](https://github.com/OpenRA/ra2/blob/9ef4a12c66258e95093bf3e651d6866382a9046d/rules/defaults.yaml#L77-L79) already defined.

(Same for `camsc11` and `camsc12`.)